docs/en/manuals/camera.md

```diff
@@ -125,7 +134,7 @@ The view matrix provided by the camera defines the position and orientation of t
 
 ### Render script
 
-Starting with Defold 1.9.6, when using the default render script Defold will automatically set the last enabled camera that should be used for rendering. Before this change, a script somewhere in the project needed to explicitly send the `use_camera_projection` message to the renderer to notify it that the view and projection from camera components should be used. This is no longer necessary, but it is still possible to do so for backwards compatibility purposes.
```

Comment:
this change was a long time ago, so it's not necessary to mention version anymore